core/dev/fhir/fhir-service.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: fhir
+  name: fhir
```

Comment:
@bradsawadye Please rename this to match the project convention ie: `fhir-service`

By having verbose naming it will help debug and understand what's going on